core/banking-protocol/src/main/resources/migration/migrations/0003-create-table-bank-profile.xml

```diff
@@ -0,0 +1,33 @@
+<databaseChangeLog logicalFilePath="db.changelog-1.0.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="create-table-bank-profile" author="mhr">
+
+        <createTable tableName="${table-prefix}bank_profile">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="${table-prefix}bank_profilePK"/>
+            </column>
+            <column name="url" type="VARCHAR(255)"/>
+            <column name="adapter_id" type="VARCHAR(255)"/>
+            <column name="idp_url" type="VARCHAR(255)"/>
+            <column name="sca_approaches" type="VARCHAR(255)"/>
+            <column name="bank_id" type="BIGINT"/>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="add-column-bank-profile" author="mhr">
+        <addColumn tableName="${table-prefix}bank">
+            <column name="bank_profile_id" type="BIGINT"/>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="add-relation-bank-to-bank-profile" author="mhr">
```

Comment:
Foreign key should typically include indexes. Also this field should probably have unique constraint if you map them as OneToOne

Reply:
done